benchmark/fs/readfile-clogging.js

```diff
@@ -0,0 +1,76 @@
+// Call fs.readFile over and over again really fast.
+// Then see how many times it got called.
+// Also report how many times we were able to run a small concurrent
+// operation on the threadpool.
+'use strict';
+
+const path = require('path');
+const common = require('../common.js');
+const filename = path.resolve(__dirname, '.removeme-benchmark-garbage');
```

Comment:
See `benchmark/fs/readfile.js`, can you include the pid?

Aside: it might be nice to use `/dev/zero` because that would remove any actual disk I/O but unfortunately there is no equivalent on Windows.  You'd still be stuck using a real file on that platform.